src/components/UI/Dropdown/Dropdown.tsx

```diff
@@ -6,27 +6,28 @@ export interface DropdownProps {
   children: React.ReactNode;
   className?: string;
   alignRight?: boolean;
-  fullWidth?: boolean;
+  shared?: string;
 }
 
 export const Dropdown: FunctionComponent<DropdownProps> = ({
   dropdownButtonText,
   children,
   className,
   alignRight,
-  fullWidth,
+  shared,
   ...props
 }) => {
   const [isOpen, setIsOpen] = useState(false);
+  const addonStylesShared = `${shared ? shared : ""}`;
+  const addonStylesButton = `${alignRight ? "ml-auto" : ""}`;
+  const addonStylesItem = `${alignRight ? "right-0" : ""}`;
```

Comment:
y does these 3 const need to be in a nested strong?

Reply:
right lol